source/com.playfab.xplatcppsdk.v141.nuspec.ejs

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+    <metadata>
+        <!-- The identifier that must be unique within the hosting gallery -->
+        <id>com.playfab.xplatcppsdk.<%- vsVer %></id>
+
+        <!-- The package version number that is used when resolving dependencies -->
+        <version><%- sdkVersion %></version>
+
+        <!-- Authors contain text that appears directly on the gallery -->
+        <authors>PlayFab</authors>
+
+        <!-- 
+            Owners are typically nuget.org identities that allow gallery
+            users to easily find other packages by the same owners.  
+        -->
+        <owners>pgilmore, toddbellMSFT</owners>
+        
+         <!-- Project URL provides a link for the gallery -->
+        <projectUrl>http://github.com/PlayFab/XPlatCppSdk</projectUrl>
+
+         <!-- License information is displayed on the gallery -->
+        <license type="expression">Apache-2.0</license>
+
+        <!-- The icon is used in Visual Studio's package manager UI -->
+        <!-- <iconUrl>https://playfab.com/assets/img/logo/azure-playfab-logo.png</iconUrl> -->
```

Comment:
We could put a legit icon here.
Microsoft logo, or PlayFab Logo. Either would work.
Make sure it's an official one, properly sized.